WalletWasabi/Wallets/WalletManager.cs

```diff
@@ -161,9 +161,9 @@ public async Task<Wallet> StartWalletAsync(Wallet wallet)
 				try
 				{
 					var cancel = CancelAllInitialization.Token;
-					Logger.LogInfo($"Starting {nameof(Wallet)}...");
+					Logger.LogInfo($"Starting Wallet {wallet.WalletName}...");
 					await wallet.StartAsync(cancel).ConfigureAwait(false);
-					Logger.LogInfo($"{nameof(Wallet)} started.");
+					Logger.LogInfo($"Wallet {wallet.WalletName} started.");
```

Comment:
```suggestion
					Logger.LogInfo($"Wallet '{wallet.WalletName}' started.");
```

Reply:
Done in https://github.com/zkSNACKs/WalletWasabi/pull/6940/commits/9258c1cabe87338f001d1b1beffad7729a023bb6